.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "10"
+  - "12"
+cache:
+  yarn: true
+script:
+  - yarn test
```

Comment:
add the lint task here ;)

Reply:
For the next PR 🍡

Reply:
ok, you can do a 'npm run lint' to test this PR ;)